File: crud/springCLI/python_entities_template/utilisateur.py

from ..datas.autocompletion import *

name = 'Utilisateur'

entity = {
    
    'Utilisateur' : {
            
        'id':{
            Type: Long,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'nom':{
            Type: String,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'prenom':{
            Type: String,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'email':{
            Type: "String",
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'motDePasse':{
            Type: "String",
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'isAdmin':{
            Type: Boolean,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'isLivreur':{
            Type: Boolean,
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'adresseList':{
            Type: "List<Adresse>",
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        'commandeList':{
            Type: "List<Commande>",
            options:{
                'dto':[
                    {'exists':True, 'fields':[]}
                ]
        
            }
        },

        }

}

File: crud/test-angular/src/app/models/utilisateur.ts

///home/hijokaidan/Documents/ProjetsCode/spring-CRUD-facilitator/crud/src/main/java/fr/cicco/crud/dto/UtilisateurDto.java
import {Adresse} from "./adresse";
import { BaseEntity } from "./base-entity";
import {Commande} from "./commande";
export interface Utilisateur extends BaseEntity{

  id:number,
  nom:string,
  prenom:string,
  email:string,
  motDePasse:string,
  isAdmin:boolean,
  isLivreur:boolean,
  adresseList:Partial<Adresse>[],
  commandeList:Commande[],

}
